src/pandapipes/diagnostic.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2020-2023 by Fraunhofer Institute for Energy Economics
+# and Energy System Technology (IEE), Kassel, and University of Kassel. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+import pandapipes as pp
+import numpy as np
+
+from pandapipes import PipeflowNotConverged
+
+try:
+    import pandaplan.core.pplog as logging
+except ImportError:
+    import logging
+
+logger = logging.getLogger(__name__)
+
+
+def check_net(net, low_length_limit_km=0.01, check_scaling_factor=1e-5):
```

Comment:
To me, the value of 0.01 km seems a bit high. I believe that this code was written when the unit was still m. So, we might rather change it to 1e-5.